src/pages/BlockDetails.vue

```diff
@@ -1,12 +1,12 @@
 <template>
   <div v-if="block && block.height">
-    <BaseContentBlock :title="$t('Block ') + '#' + block.height" class="block-details-card">
+    <BaseContentBlock :title="$t('blocks',2) + '#' + block.height" class="block-details-card">
```

Comment:
1. To use pluralisation feature, you need to use `$tc`, not `$t`.
2. `$t(..) + '#' + block.height` is not how `$t` should be used. It is designed with interpolation in mind. Instead, you should use:
   ```js
   $t('foo.bar', { height: block.height })
   // where `foo.bar` is something like "Block #{height}"

Re-read [syntax](https://vue-i18n.intlify.dev/guide/essentials/syntax.html) and [pluralisation](https://vue-i18n.intlify.dev/guide/essentials/pluralization.html) topics and combine both of my points.

Reply:
1. Researched about it `$tc`  is not needed with `composition API` as `$t` supports pluralisation. 
References: [docs](https://arc.net/l/quote/brapfxsi) , [stack overflow](https://arc.net/l/quote/xpiopjne)
2. Acknowledged and committed 👍🏼 
3. For string concatination (Two approaches are their ) : 
Example: 
```
en.json => "parent" : "Parent",
usage=> `${$t("blockDetails.parent")} ${$t("block", 1)} ${$t("hash")}`
or => {{ $t("blockDetails.parent") + " " +$t("blocks", 2) +" " +$t("hash") }}     
output=> Parent Block Hash

```
```
en.json => "parent" : "Parent Block Hash",
usage=>  {{ $t("blockDetails.parent") }} 
output=> Parent Block Hash
```
Kindly , suggest which approach to follow.

Reply:
Try using the following format in messages: `Block #{height}`